### PR TITLE
Fix issue in Tasks list and fix wrong request in Sales Record

### PR DESCRIPTION
**Pull request by MikeVitik**

### CODE REVIEW

File: packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <div id="task-form" class="form-compact">
   <dx-toolbar>
-    <dxi-item location="before"><span class="form-title">Details</span></dxi-item>
+    <dxi-item location="before"
+      ><span class="form-title">Details</span></dxi-item
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.html:4]: Add linebreak after </span>

File: packages/angular/src/app/pages/planning-task-list/task-list-kanban/task-list-kanban.component.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import {
-  Component, OnInit, NgModule, Input, SimpleChanges, OnChanges,
+  Component, NgModule, Input, SimpleChanges, OnChanges,
 } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { DxButtonModule } from 'devextreme-angular/ui/button';
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-task-list/task-list-kanban/task-list-kanban.component.ts:5]: Remove DxLoadPanelModule from this component

File: packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.html (hunk 1 above)
Comment (MikeVitik) [reply packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.html:4]: Postpone to prettier integration

File: packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.html [hunk 3]
```diff
@@ -97,32 +101,36 @@
...
       </dxi-item>
-
     </dxi-item>
 
     <dxi-item [colSpan]="2">
-      <edit-view-item *ngIf="!isEditing"
-                      [isEditing]="isEditing"
-                      label="Details"
-                      [value]="task.description"
+      <edit-view-item
```
Comment (GoodDayForSurf) [inline packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.html:123]: for me, this less readable than previous variant, and it increase rows in file

File: packages/angular/src/app/shared/services/rwa.service.ts [hunk 4]
```diff
@@ -49,47 +51,47 @@ export class RwaService {
...
-
-    public getSales = (startDate: string, endDate: string) => this.http
-      .get<any>(`${API_URL}/Analytics/SalesAdaptiveGrouped/${startDate}/${endDate}`);
+  public getSalesByStateAndCity = (startDate: string, endDate: string) => this.http
+    .get(`${API_URL}/Analytics/SalesByStateAndCity/${startDate}/${endDate}`);
+
+  public getSalesByState = (data) => {
+    let dataByState;
+    from(data)
+      .pipe(
```
Comment (GoodDayForSurf) [inline packages/angular/src/app/shared/services/rwa.service.ts:60]: using reduce() more simple than chain rx operators

**Review by GoodDayForSurf — CHANGES_REQUESTED** [review]

File: packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.html (hunk 3 above)
Comment (MikeVitik) [reply packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.html:123]: It's about linter config.

**Review by Shuvaevmedia — APPROVED** [review]

**Review by dxvladislavvolkov — APPROVED** [review]

**Review by GoodDayForSurf — APPROVED** [review]

**Pull request merged**